viewer/packages/360-images/src/collection/DefaultImage360Collection.ts

```diff
@@ -104,7 +119,7 @@ export class DefaultImage360Collection implements Image360Collection {
   }
 
   public setIconsVisibility(visible: boolean): void {
```

Comment:
```suggestion
  private setIconsVisibility(visible: boolean): void {
```
This shouldn't be used outside this class anymore, right?

Reply:
Recombined the functions. We now only have exposed setIconsVisibility.